src/reynir/matcher.py

```diff
@@ -1468,7 +1590,7 @@ def span(self):
     @property
     def nouns(self):
         """ Returns the lemmas of all nouns in the subtree """
-        return self._list(lambda t: t.tcat == "no" or t._cat in _GENDERS)
+        return self._list(lambda t: t.tcat == "no" or t._cat in _GENDERS or t.tcat == "entity")
```

Comment:
Better: `self._list(lambda t: t.tcat == "no" or t.tcat == "entity" or t._cat in _GENDERS)`
or: `self._list(lambda t: t.tcat in {"no", "entity"} or t._cat in _GENDERS)`